pl_examples/basic_examples/cnn_cifar_pipe.py

```diff
@@ -0,0 +1,244 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+from argparse import ArgumentParser
+
+import torch
+import torch.distributed as torch_distrib
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+from torch import optim
+
+import pytorch_lightning as pl
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import LearningRateMonitor, ModelCheckpoint
+from pytorch_lightning.metrics.functional import accuracy
+from pytorch_lightning.plugins.pipe_rpc_plugin import FAIRSCALE_AVAILABLE, PipeRpcPlugin
+from pytorch_lightning.utilities import BOLT_AVAILABLE
+
+if BOLT_AVAILABLE:
+    import pl_bolts
+    from pl_bolts.transforms.dataset_normalizations import cifar10_normalization
+
+
+"""
+To run with Pipe
+python benchmarks/cnn_cifar_pipe.py --use_pipe 1
```

Comment:
Should be `python pl_examples.basic_examples/cnn_cifar_pipe.py`

imo the added complexity of having both normal training and pipe in one script is a bit weird. I understand that you need it for debugging purposes, but I think for final release, it should just be an example for pipe!

Reply:
I am not sure. I think it helps to understand what pipe does.